### DISCUSSION TITLE
how to custom font?

### DISCUSSION THREAD

**Discussion by fiqryq** [timlrx/tailwind-nextjs-starter-blog#320] [Q&A]
1. How to custom font?
2. How to change sytax hilight background color?
<img width="1099" alt="Screen Shot 2022-01-02 at 16 00 11" src="https://user-images.githubusercontent.com/25787603/147871114-470e9b8e-0e13-49f9-8d7a-51d304ec3e0e.png">

**Comment by timlrx** [comment]
1. Change [google fonts](https://github.com/timlrx/tailwind-nextjs-starter-blog/blob/master/pages/_document.js#L27) and update [tailwind config](https://github.com/timlrx/tailwind-nextjs-starter-blog/blob/master/tailwind.config.js#L19)
2. That should correspond to `backgroundColor` for `pre` tags - [tailwind config](https://github.com/timlrx/tailwind-nextjs-starter-blog/blob/master/tailwind.config.js#L54). You can customise it for light theme and dark theme.

**Reply by fiqryq** [reply]
and how about change text size inside mdx?
![Screen Shot 2022-01-02 at 17 31 07](https://user-images.githubusercontent.com/25787603/147873057-1fc20acb-ecf6-472d-9d24-0f5e45140125.png)

**Reply by timlrx** [reply]
Check out the [tailwind docs on the typography plugin](https://tailwindcss.com/docs/typography-plugin) and modify the config file or template accordingly.
